client/components/enrich/SettingsJSONL.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div>
+    <Select
+      :items="unpaywallAttr"
+      :label="'simple'"
+      @tab="getTab"
+    />
+    <Select
+      :items="oa_location"
+      :label="'best_oa_location'"
+      @tab="getTab"
+    />
+    <Select
+      :items="oa_location"
+      :label="'first_oa_location'"
+      @tab="getTab"
+    />
+    <Select
+      :items="oa_location"
+      :label="'oa_locations'"
+      @tab="getTab"
+    />
+  </div>
+</template>
+<script>
+import Select from '~/components/enrich/Select.vue'
+
+export default {
+  components: {
+    Select
```

Comment:
HTML tags are case-insensitive, maybe you should rename this component to avoid conflicts with the native `select` tag.

Reply:
rename this by SelectAttributes